library/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildToolsVersion '25.0.0'
```

Comment:
buildToolsVersion is already definded on the file, try to follow the same schema